tests/cupy_tests/cuda_tests/test_nccl.py

```diff
@@ -19,60 +20,60 @@ class TestNCCL(unittest.TestCase):
 
     @testing.gpu
     def test_single_proc_ring(self):
-        id = cuda.nccl.get_unique_id()
-        comm = cuda.nccl.NcclCommunicator(1, id, 0)
+        id = nccl.get_unique_id()
```

Comment:
FYI (no need to revert this), you didn't have to change `cuda.nccl` to `nccl` just by adding `import cupy.cuda.nccl` 😄